[MOL-14046][HFAH] auto uppercase IC input field for eWills & HDB carparking (#326)
LifeSG/react-design-system: stories/form/form-masked-input/form-masked-input.stories.mdx

                    value="S1234567D"
                    maskRange={[2, 5]}
                />
                <Form.MaskedInput

>                     label="This is the mask field with uppercase input"

Reviewer: ```suggestion
                    label="This is set to be transformed to uppercase"
```

                    maskRange={[2, 5]}
                    transformInput={"uppercase"}
                />
                <Form.MaskedInput

>                     label="This is the mask field with lowercase input"

Reviewer: ```suggestion
                    label="This is set to be transformed to lowercase"
```
